scripts/globals/mobskills/Fluid_Toss_Claret.lua

```diff
@@ -0,0 +1,41 @@
+---------------------------------------------
+--  Fluid Toss
+--
+--  Description: Lobs a ball of liquid at a single target.
+--  Type: Ranged
+--  Utsusemi/Blink absorb: 1 shadow
+--  Range: Unknown range
+---------------------------------------------
+
+require("scripts/globals/settings");
+require("scripts/globals/status");
+require("scripts/globals/monstertpmoves");
+
+---------------------------------------------
+
+function onMobSkillCheck(target,mob,skill)
+	return 0;
+end;
+
+function onMobWeaponSkill(target, mob, skill)
+
+	local numhits = 1;
+	local accmod = 1;
+	local dmgmod = 5.5;
+	local info = MobPhysicalMove(mob,target,skill,numhits,accmod,dmgmod,TP_DMG_VARIES,1,2,3);
+	local dmg = MobFinalAdjustments(info.dmg,mob,skill,target,MOBSKILL_PHYSICAL,MOBPARAM_SLASH,info.hitslanded);
+	target:delHP(dmg);
+    
+    -- Apply poison
+    if (target:hasStatusEffect(EFFECT_POISON) == false) then
+        target:addStatusEffect(EFFECT_POISON, 100, 3, math.random(3,6) * 3); -- Poison for 3-6 ticks.
+    else
+        if (target:hasStatusEffect(EFFECT_POISON) == true) then
+            if (target:getStatusEffect(EFFECT_POISON):getPower() < 100) then
+                target:addStatusEffect(EFFECT_POISON, 100, 3, math.random(3,6) * 3); -- Poison for 3-6 ticks.                  

```

Comment:
You have to remove the poison first then overwrite. Poison will not overwrite itself